lib/service/connect.go

```diff
@@ -684,6 +687,108 @@ func (process *TeleportProcess) firstTimeConnect(role types.SystemRole) (*Connec
 	return connector, nil
 }
 
+func (process *TeleportProcess) initOpenSSH() {
+	process.RegisterWithAuthServer(types.RoleNode, SSHIdentityEvent)
+	process.SSHD = openssh.NewSSHD(process.Config.OpenSSH.RestartCommand)
+	process.RegisterCriticalFunc("common.rotate_openssh2", process.syncOpenSSHRotationState)
```

Comment:
Reasoning behind the "common" prefix here? I would expect something more like `"openssh2.rotate"` based on other names.